content/_data/upgrades/2-263-2.adoc

```diff
@@ -111,4 +111,9 @@ Given how labels are stored on disk, some label names would allow overriding unr
 
 This is not considered to be a security vulnerability as Overall/Administer permission is required to save label configurations, and users with that permission can use the script console and have access to the Jenkins controller file system anyway.
 
+Some plugins (like Mesos cloud) allow unsafe characters (like ':') in the labels of the agents they create.
+Those plugins may fail to provision an agent due to the unsafe character in the agent label as reported in jira:JENKINS-65169[].
+It is much better to change the label convention in the plugin configuration to avoid unsafe characters.
+If that is not possible, administrators may consider disabling this hardening.
```

Comment:
This part should not be in this section as it is about something entirely different.

We did not include a section applying to JENKINS-65169/SECURITY-2021 and need a new one.